### ISSUE TITLE
FedML installation of AWS EC2 Ubuntu Instance

### ISSUE THREAD

**Issue by pranavgundewar** [FedML-AI/FedML#221]
Hi!

I am trying to install fedml on AWS EC2 instances. I am facing this issue when I tried pip install fedml-

![image](https://user-images.githubusercontent.com/37206432/167967162-ad2881cb-15a2-4e04-97ea-c36c908e9a04.png)

Please let me know how to install fedml. Thank you.

**Comment by khoaanhtran** [comment]
Did you used Python ver 3.10? I have tested with this version and got an error the same with you.

Remove it and install below 3.8 or 3.7, that's fine :)

Hope help you

**Comment by chaoyanghe** [comment]
@pranavgundewar @khoaanhtran yeah, we are trying to migrate it to Python in the next iteration. Please try 3.8 or 3.7 first.

**Comment by pranavgundewar** [comment]
Hi @chaoyanghe @khoaanhtran Thank you for the suggestions. It worked for me.

I am facing one more question. I have 3 EC2 instanced running on AWS. 1 is for a server and another 2 are for 2 separate clients each with 1 GPU. How to create GPU mapping and config for such an arrangement?

Please let me know. Thank you.

**Comment by chaoyanghe** [comment]
@pranavgundewar have you solved your issue? You can configure your GPU mapping in each silo separately.

**Comment by fedml-dimitris** [comment]
@pranavgundewar were you able to resolve your issue?
